modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/exception/CommerceWorkflowException.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author Mahmoud Azzam
+ */
+public class CommerceWorkflowException extends PortalException {
+
+	public CommerceWorkflowException() {
```

Comment:
usually exception are added to the service.xml and then build with the service builder tool